sse.go

```diff
@@ -27,6 +27,8 @@ type Streamer struct {
 	connecting    chan client
 	disconnecting chan client
 	bufSize       uint
+	
+	CORS bool
```

Comment:
Shouldn't this be rather a `string`, with the CORS header being set whenever the string is not empty?
This would allow more fine-grained control than setting `*` all the time.